### ISSUE TITLE
`choose-your-xml-rs` is deprecated. Now what?

### ISSUE THREAD

**Issue by ErichDonGubler** [not-yet-awesome-rust/not-yet-awesome-rust#53]

**Comment by ErichDonGubler** [comment]
@razrfalcon might be able to help here...

**Comment by RazrFalcon** [comment]
https://github.com/RazrFalcon/roxmltree#alternatives ?

**Comment by jonathan-s** [comment]
Roxmltree seems to fulfill most criterias in the xml section?

**Comment by crlf0710** [comment]
Not really. roxmltree doesn't support dtd, doesn't support writing xml, let alone other xml functionalities like XPath XSLT, XQuery, XSD etc .

**Comment by RazrFalcon** [comment]
>roxmltree doesn't support dtd

It has a partial DTD support. 

>XPath XSLT, XQuery, XSD

This is not a part of XML tree library.

**Comment by crlf0710** [comment]
@RazrFalcon true, but it doesn't 
> fulfill most criterias in the xml section

.

**Comment by crlf0710** [comment]
I think `roxmltree` should be mentioned non-the-less in the section. However we still many crates to fulfill the more "advanced" parts of xml proessing. This can ease the parsing and validating of xml-based formats, since those formats will not go away any time soon.
